src/renderers/DomRenderer.js

```diff
@@ -248,6 +248,23 @@ function insertStyle(style: HTMLElement, options: PriorityOptions) {
   getHead().insertBefore(style, prevNode)
 }
 
+/**
+ * Read jss settings from the page if the user has set them.
+ */
+function getSettingsFromInlineJSON() : Object {
+  let config = {}
+  const inlineJsonBlock = document.querySelector('script[type="application/json"][data-jss-selector="jss-settings-json"]')
```

Comment:
this query can be simplified to an id or simple class name, right?

Reply:
could simplify to `#jss-settings-json` if prefered?